src/environmentd/src/lib.rs

```diff
@@ -362,7 +360,37 @@ impl Listeners {
         )
         .await?;
 
+        // Determine whether we should perform a 0dt deployment.
+        let enable_0dt_deployment = {
+            let default = config
+                .system_parameter_defaults
+                .get(ENABLE_0DT_DEPLOYMENT.name())
+                .map(|x| {
+                    strconv::parse_bool(x).map_err(|err| {
+                        anyhow!(
+                            "failed to parse default for {}: {}",
+                            ENABLE_0DT_DEPLOYMENT.name(),
+                            err
+                        )
+                    })
+                })
+                .transpose()?;
+            let ld = get_ld_value("enable_0dt_deployment", &remote_system_parameters, |x| {
+                strconv::parse_bool(x).map_err(|x| x.to_string())
+            })?;
+            let catalog = openable_adapter_storage.get_enable_0dt_deployment().await?;
+            let computed = default.or(ld).or(catalog).unwrap_or(false);
+            info!(
+                "enable_0dt_deployment value of {} computed from default {:?} catalog {:?} LD {:?}",
```

Comment:
Nit, the log would be easier to read if it was in order of priority, i.e. `"enable_0dt_deployment value of {} computed from default {:?} LD {:?} catalog {:?}",`

Reply:
Oops! I got the log message right but the computation wrong. Thanks for catching this. Fixed, and also improved the log message formatting a bit, and also applied that same formatting improvement to the analogous persist_txn_tables log message.